alert/teams.go

```diff
@@ -88,11 +90,35 @@ func SendCard(webhook string, card *TeamsMessageCard) (*http.Response, error) {
 	if err := json.NewEncoder(buffer).Encode(card); err != nil {
 		return nil, fmt.Errorf("Failed encoding message card: %v", err)
 	}
-	res, err := http.Post(webhook, "application/json", buffer)
+
+	c := &http.Client{
+		Transport: &http.Transport{
+			Proxy: http.ProxyFromEnvironment,
+			DialContext: (&net.Dialer{
+				Timeout:   30 * time.Second,
+				KeepAlive: 30 * time.Second,
+				DualStack: true,
+			}).DialContext,
+			MaxIdleConns:          100,
+			IdleConnTimeout:       90 * time.Second,
+			TLSHandshakeTimeout:   30 * time.Second,
```

Comment:
It might be better to use flags to configure the timeout.

It can be declared from here:

https://github.com/bzon/prometheus-msteams/blob/a6f4276a2ad3289786403e9be4c3fd8a3590d18e/cmd/server.go#L78-L92

and pass it here:

https://github.com/bzon/prometheus-msteams/blob/a6f4276a2ad3289786403e9be4c3fd8a3590d18e/cmd/server.go#L202-L207

https://github.com/bzon/prometheus-msteams/blob/a6f4276a2ad3289786403e9be4c3fd8a3590d18e/alert/prom_handler.go#L36-L45

then pass the **timeout configs** here:

https://github.com/bzon/prometheus-msteams/blob/a6f4276a2ad3289786403e9be4c3fd8a3590d18e/alert/teams.go#L189

Reply:
This should be then make also configurable via values.yaml for Helm chart.